### PR TITLE
feat: Add prometheus in docker compose to inspect metrics

### PR DESCRIPTION
**Pull request by Matovidlo**
Jira: [PSGO-854](https://keboola.atlassian.net/browse/PSGO-854)

**Changes:**
- Add prometheus container into docker compose
- Adjust documentation to be able to retrieve metrics using prometheus and services
- Change port numbers for metrics scraping to be set same as in docker-compose.yml for development environment.

-----------

[PSGO-854]: https://keboola.atlassian.net/browse/PSGO-854?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: provisioning/dev/prometheus/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+       - 'prometheus:9090'
+ - job_name: templatesAPI
+   static_configs:
+    - targets:
+       - 'dev:9000'
+ - job_name: streamAPI
+   static_configs:
+    - targets:
+       - 'dev:9001'
+ - job_name: AppsProxy
```
Comment (hosekpeter) [inline provisioning/dev/prometheus/prometheus.yml:16]: why not 'appsProxy' ?

**Comment by hosekpeter** [comment]
Why isn't air.toml file changed for Template API? 🤔

File: docs/development.md [hunk 2]
```diff
@@ -37,6 +37,26 @@ Start an interactive console in a container, run:
 docker compose run --rm -u "$UID:$GID" --service-ports dev bash
 ```
 
+### Start Dev Container inspecting metrics
+
+When we would like to inspect metrics of our services, there have to be made adjustement under `dev` service.
```
Comment (jachym-tousek-keboola) [inline docs/development.md:42]: Maybe you can add it into docker compose, comment it out with a note what it does / when to enable it?

File: provisioning/dev/prometheus/prometheus.yml (hunk 1 above)
Comment (Matovidlo) [reply provisioning/dev/prometheus/prometheus.yml:16]: It's just name of job in prometheus, can be anything, not very important but I will change it

**Comment by Matovidlo** [comment]
@hosekpeter The templates API have default `:9000` port and it is available, I can adjust the configuration but it is default value.

File: docs/development.md (hunk 2 above)
Comment (Matovidlo) [reply docs/development.md:42]: I will leave this in documentation and add commented out code with explanation.

File: provisioning/dev/prometheus/prometheus.yml (hunk 1 above)
Comment (Matovidlo) [reply provisioning/dev/prometheus/prometheus.yml]: aa81117023dea3abdf4eb9fdc29412d28460c402

**Review by hosekpeter — APPROVED** [review]
LGTM

**Review by jachym-tousek-keboola — APPROVED** [review]

**Pull request merged**